test/e2e/case20_delete_objects_test.go

```diff
@@ -445,7 +445,7 @@ var _ = Describe("Test Object deletion", Ordered, func() {
 					case20ConfigPolicyNameChange, testNamespace, false, defaultTimeoutSeconds)
 
 				return managedPlc
-			}, defaultTimeoutSeconds, 1).Should(BeNil())
+			}, 30, 1).Should(BeNil())
```

Comment:
I wonder if we should have a separate setting like `defaultConsistentlyDuration`. It's not *necessary*, but this PR makes me feel like it could be nice.

Reply:
`defaultConsistentlyDuration` good idea